# fix compiler warning about an empty body in if statement. Obviously,
# the colon is wrong, as it makes the function to return unconditionally,
# even though there are other conditional returns following below
# please CHECK this patch soroughly in context

--- a/rosetta++/elliptic_msd.cc	2008-04-17 22:27:00.000000000 +0200
+++ b/rosetta++/elliptic_msd.cc	2008-12-03 15:18:28.000000000 +0100
@@ -227,7 +227,8 @@
 
 //$$$      calc_msd_best(msd_start,msd_start+msd_atoms/5,frag2_cost);
 
-	if (total_begin < 1); return //warning sign!
+	if (total_begin < 1) 
+	return //warning sign!
 	add_fast_wobble(double_begin,double_end-double_begin+1,2,0,rama_score,
 	 dist_score,gfrag,total_begin,total_end);
 
